src/components/member-profile/contribution/index.js

```diff
@@ -5,6 +5,8 @@ import classNames from '@components/member-profile/contribution/contribution.mod
 import PRLink from '@components/member-profile/contribution/pr-link';
 import { timeWas } from '@helper-functions/time-was';
 import { HOST_NAME } from '@constants/AppConstants';
+import { useState } from 'react';
+import SuperUserOptions from '../super-user-options/super-user-options-container';
 
 const renderPRLinks = (prList) =>
   prList.map(({ url }, index) => {
```

Comment:
@vinayak-trivedi  We usually don't pass index as a key, you can check here -
https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66

Reply:
okay this is not a change that I've made abhay, I agree this need to be refactored, this is a concern of another PR